xpdacq/beamtimeSetup.py

```diff
@@ -259,25 +258,33 @@ def _load_bt_info(bt_obj, required_fields):
 def _tar_user_data(archive_name, root_dir=None, archive_format='tar'):
     """ Create a remote tarball of all user folders under xpdUser directory
     """
-    archive_full_name = os.path.join(glbl_dict['archive_dir'],
-                                     archive_name)
+    remote_archive_name = os.path.join(glbl_dict['archive_dir'],
+                                       archive_name)
     if root_dir is None:
         root_dir = glbl_dict['base']
     try:
         os.chdir(root_dir)
+        # rename xpdUser to <xpdUser_archivefallname>
+        src = glbl_dict['home']
+        local_archive_name = '_'.join([src, archive_name])
+        local_archive_name = shutil.move(src, local_archive_name)
```

Comment:
I don't understand why this is done recursively.  why is it ``local_archive_name = shutil.move(src, local_archive_name)``.  What does ``shutil.move`` return?  Also I am not sure what is in ``glbl_dict['home']``. I suppose that is ``xpdUser``?  Or is it `~`. If the former, the line above seems to suggest that you are putting the archive into ``xpdUser`` which doesn't seem to be right.

Reply:
`local_archive_name` on the left side (new) is the return value from the `shutil.move` function which tells us where is the file actually moved to, with respect to the file system. 

I overwrite the `local_archive_name` (old) on the right side with the new one just in case if `xpdUser` is placed at some weird location in the future (and possible symlic back), `shutil` can still help us figure out the absolute path on the local file system.

Yes, `glbl['home']` is `<nfs_mount_path>/xpdUser`. I am renaming `xpdUser`, loaded with user data to `xpdUser_<user beamtime_indentifier>` so that the next beamtime won't be blocked by the archiving process.

Reply:
OK, this makes sense.  But please check what the archive will be called.  You are wanting ``xpdUser_<user beamtime id>`` but based on what you have written above, you may end up with ``~\this_and_that\xpdUser_<user beamtime id>``